src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,49 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+    private final int NO_VALUE_RESPONSE = 0;
+    public int splitAndSum(String input) {
```

Comment:
멤버 변수와 메소드간 개행으로 구분하는 것이 자바 컨벤션이므로 지키는 게 좋을 것 같아요
```suggestion
    private final int NO_VALUE_RESPONSE = 0;
    
    public int splitAndSum(String input) {
```

Reply:
넵 참고하겠습니다!